src/commands/DensityProfile.cpp

```diff
@@ -80,9 +108,69 @@ Averager<double> DensityProfile::setup(int argc, const char* argv[]) {
 
     AveCommand::parse_options(args);
 
-    options_.npoints = string2long(args.at("--points").asString());
-    options_.selection = args.at("--selection").asString();
+    n_axis_ = 0;
 
+    if (args.at("--axis")) {
+        if (args.at("--axis").asStringList().size() == 0) {
+        } else if (args.at("--axis").asStringList().size() == 1) {
+            n_axis_ ++;
+            axis_x_ = axis_parse(args.at("--axis").asStringList()[0]);
+            options_.type_profile[n_axis_ - 1] = 1;
+        } else if (args.at("--axis").asStringList().size() == 2) {
+            n_axis_ ++;
+            options_.type_profile[n_axis_ - 1] = 1;
+            axis_x_ = axis_parse(args.at("--axis").asStringList()[0]);
+            n_axis_ ++;
+            options_.type_profile[n_axis_ - 1] = 1;
+            axis_y_ = axis_parse(args.at("--axis").asStringList()[1]);
+        } else {
+            throw CFilesError("Too many axis were given");
+        }
+    }
+
+    if (args.at("--radial")) {
+        if (args.at("--radial").asStringList().size() == 0) {
+        } else if (args.at("--radial").asStringList().size() == 1) {
+            n_axis_ ++;
+            if (n_axis_ == 1) {
+                axis_x_ = axis_parse(args.at("--radial").asStringList()[0]);
+            } else if (n_axis_ == 2) {
+                axis_y_ = axis_parse(args.at("--radial").asStringList()[0]);
+            } else { 
+                throw CFilesError("Too many axis were given");
+            }
+            options_.type_profile[n_axis_ - 1] = 2;
+        } else if (args.at("--radial").asStringList().size() == 2) {
+            if (n_axis_ == 2) {
+                throw CFilesError("Too many axis were given");
+            } else {
+                n_axis_ ++;
+                options_.type_profile[n_axis_ - 1] = 2;
+                axis_x_ = axis_parse(args.at("--radial").asStringList()[0]);
+                n_axis_ ++;
+                options_.type_profile[n_axis_ - 1] = 2;
+                axis_y_ = axis_parse(args.at("--radial").asStringList()[0]);
+            }
+        }
+    }
+
```

Comment:
Also, the code do not looks nice here =/ How about collecting everything in a `std::vector<std::pairs<Axis, AxisType>>`, and checking at the end that we have 1 or 2 axis only?